lib/ethpy/ethpy/hyperdrive/interface.py

```diff
@@ -116,9 +116,9 @@ def get_hyperdrive_config(hyperdrive_contract: Contract) -> dict[str, Any]:
     # Ok so, the contracts store the time stretch constant in an inverted manner from the python.
     # In order to not break the world, we save the contract version as 'invTimeStretch' and invert
     # that to get the python version 'timeStretch'
-    # TODO invTimeStretch should be in fixed point notation
-    pool_config["invTimeStretch"] = hyperdrive_config["timeStretch"]
-    pool_config["timeStretch"] = FixedPoint(1) / FixedPoint(scaled_value=hyperdrive_config["timeStretch"])
+    # TODO fix this
```

Comment:
fix what? Can you clarify so that this TODO is helpful to a reader at a later time?

Reply:
The comment about this is right above this line, but I'll change it